lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystem.cs

```diff
@@ -127,7 +127,66 @@ public void TestDeleteDirectoryRecursive()
         public void TestCreateUriForPath()
         {
             var testContext = new TestContext();
-            Assert.Equal(FakeUri, testContext.GetAzureFileSystem().CreateUriForPath(FakeUri.LocalPath));
+            var tests = new[]
```

Comment:
nit: IMO, it's better to make individual tests instead of creating these loops. That way the test runner can see which individual test is failing.

Reply:
Done.